tests/unit-tests/file_tests/non-deterministic/clock_gettime_highlevel.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <time.h>
+
+/*
+    This is higher level test for clock_gettime, which we don't directly call clock_gettime function but still call this syscall eventually
```

Comment:
This comment could be better, like explaining that clock() calls clock gettime, and briefly say what the test does.